src/components/Reports/FinancialAccountsReport/Header/TransactionsHeader.test.tsx

```diff
@@ -8,86 +8,85 @@ import { SnackbarProvider } from 'notistack';
 import { I18nextProvider } from 'react-i18next';
 import TestRouter from '__tests__/util/TestRouter';
 import { GqlMockedProvider } from '__tests__/util/graphqlMocking';
-import { FinancialAccountsWrapper } from 'pages/accountLists/[accountListId]/reports/financialAccounts/[financialAccountId]/Wrapper';
+import { UrlFiltersProvider } from 'src/components/common/UrlFiltersProvider/UrlFiltersProvider';
 import i18n from 'src/lib/i18n';
 import theme from 'src/theme';
 import { FinancialAccountQuery } from '../Context/FinancialAccount.generated';
-import {
-  FinancialAccountContext,
-  FinancialAccountType,
-} from '../Context/FinancialAccountsContext';
+import { FinancialAccountContext } from '../Context/FinancialAccountsContext';
 import { defaultFinancialAccount } from './HeaderMocks';
 import { TransactionsHeader } from './TransactionsHeader';
 
 const accountListId = 'account-list-1';
 const financialAccountId = 'financialAccountId';
 const handleNavListToggle = jest.fn();
-const defaultHasActiveFilters = false;
 const handleFilterListToggle = jest.fn();
-const defaultSearchTerm = '';
-const setSearchTerm = jest.fn();
 const handleExportCSV = jest.fn();
 
-const router = {
-  query: { accountListId },
-  isReady: true,
-};
-
 interface ComponentsProps {
   disableExportCSV?: boolean;
-  hasActiveFilters?: boolean;
-  searchTerm?: string;
+  isReady?: boolean;
 }
+
 const Components = ({
-  hasActiveFilters = defaultHasActiveFilters,
-  searchTerm = defaultSearchTerm,
   disableExportCSV = false,
-}: ComponentsProps) => (
-  <I18nextProvider i18n={i18n}>
-    <LocalizationProvider dateAdapter={AdapterLuxon}>
-      <SnackbarProvider>
-        <ThemeProvider theme={theme}>
-          <TestRouter router={router}>
-            <GqlMockedProvider<{
-              FinancialAccount: FinancialAccountQuery;
-            }>
-              mocks={{
-                FinancialAccount: defaultFinancialAccount,
+  isReady = true,
+}: ComponentsProps) => {
+  return (
+    <I18nextProvider i18n={i18n}>
+      <LocalizationProvider dateAdapter={AdapterLuxon}>
+        <SnackbarProvider>
+          <ThemeProvider theme={theme}>
+            <TestRouter
+              router={{
+                query: { accountListId },
+                isReady: isReady,
```

Comment:
`isReady` isn't set by any tests. Can we remove this prop?

```suggestion
                isReady: true,
```

Reply:
Yeah, let's remove it to keep it simple. If we ever end up needing it, then we can update `TestComponent` to be more flexible.